.github/workflows/ci.yml

```
@@ -595,8 +531,7 @@ jobs:
         run: |
           if [[ "${{ needs.deploy-rustdoc.result }}" != "success" ||
                 "${{ needs.build-linux-x86_64-post-merge.result }}" != "success" ||
-                "${{ needs.upload-chain-specs.result }}" != "success" ||
-                "${{ needs.local-env-post-merge.result }}" != "success" ]]
+                "${{ needs.upload-chain-specs.result }}" != "success" ]]
```

looks like we close if too soon
```suggestion
if [[ "${{ needs.deploy-rustdoc.result }}" != "success" ||
                "${{ needs.build-linux-x86_64-post-merge.result }}" != "success" ||
                "${{ needs.upload-chain-specs.result }}" != "success" ||
                "${{ needs.deploy-ci-preview.result }}" != "success" ||
                "${{ needs.upload-to-s3.result }}" != "success" ]]; then
```